src/main/java/org/assertj/core/api/AbstractDateAssert.java

```diff
@@ -2188,6 +2188,51 @@ public S hasSameTimeAs(Date date) {
   }
 
   /**
+   * Same assertion as {@link #hasSameTimeAs(Date)} but given date is represented as String either with one of the
+   * supported defaults date format or a user custom date format (set with method {@link #withDateFormat(DateFormat)}).
+   * <p/>
+   * Beware that the default formats are expressed in the current local timezone.
+   * <p/>
+   * Example:
+   *
+   * <pre><code class='java'>
+   * // assertion will pass
```

Comment:
the code examples are not helpful if you don't define date, moreover I'm pretty sure that one the "succeeding" assertion fails.